src/components/Tag.tsx

```diff
@@ -0,0 +1,36 @@
+import PropTypes from "prop-types";
+import classnames from "classnames";
+import React from "react";
+
+import "govuk-frontend/govuk/components/tag/_tag.scss";
```

Comment:
Just to confirm: `lbh-frontend` has no tag specific styles, right?

Reply:
The tag specific styles for `lbh-frontend` are defined via phase-banner. The tag used on its own has no lbh styles, probably because there was no reason to do so yet, but I feel it may be something that appears when we get to [task lists](https://design-system.service.gov.uk/patterns/task-list-pages/) (verified / done stamps)  . Might be worth raising this to @LBHELewis  because at the moment, a tag called outside the phase banner will look different stylistically to a tag called within it.

Reply:
This is related to LBHackney-IT/LBH-frontent#35.